src/client/index.ts

```diff
@@ -1,12 +1,73 @@
 /**
- * @module Index
+ * OpenFin Notifications provide developers with a uniform way to create, display and organize desktop notifications
+ * as well as responding to notification events.
+ *
+ * Notifications will be displayed as toasts as well as being listed and organized in a Notification Center. The
```

Comment:
True, but clarifying that feels like it would be a bit verbose for something this "front and centre" within the docs.

Once we make the center close on blur, this will also be a much less likely occurrence (though still one we should be handling).

Reply:
Yea, as I said, too much of a technicality to be worth going into detail, but think we should keep it in mind so we don't mislead too much.